esi_ui/static/dashboard/esi/lessee/offers/esi-offers-table.controller.js

```diff
@@ -69,6 +72,20 @@
         });
       });
     }
+
+    function getOfferAvailabilityStatusColor(status) {
```

Comment:
maybe just `getOfferStatusColor`, since we're not looking at availability at all here